### DISCUSSION TITLE
Allow changing sort order of release assets

### DISCUSSION THREAD

**Discussion by tycho-kirchner** [community/community#6603] [Other Feature Feedback, Questions, & Ideas] [answered]
Apparently released assets are sorted by name, which is not always an optimal choice. When several editions are offered, many people just use the top one, however, in below project users are strongly encouraged to install the bottom-most edition ( shournal_2.5_amd64.deb ). On the other hand I do not want to change the package names, as those are determined by the Debian packaging standard. So I would kindly ask for an option to manually change the order of the assets. 

Thanks and kind regards

Tycho Kirchner
![Screenshot_20211022_085854](https://user-images.githubusercontent.com/29805132/138407931-2f5fac80-5fbc-4fd0-95c2-e1333578a4d0.png)

**Comment by MylesBorins** [comment, marked as answer]
hey @tycho-kirchner. I am scoping a a number of features that would improve the overall experience of downloading assets, because as you have pointed out there is a lot of room for improvement.

I personally think that we are more likely to ship something that makes it VERY easy to download the correct asset for the platform you are on (like https://github.com/github/feedback/discussions/6163). I don't think that sort order is something that would be manually managed on a per release (and per asset) basis though.

So I'm going to mark this specific discussion as "answered" as we are not going to likely explore manual ordering, but I can assure you improving the overall asset experience is top of mind for the team

**Reply by tycho-kirchner** [reply]
Hi @MylesBorins. It's been a while, but the issue persists. As you can see [here](https://github.com/tycho-kirchner/shournal/releases/tag/v2.9) `shournal-docker_2.9_amd64.deb` is still on top, although most users probably want to download `shournal_2.9_amd64.deb`. Please note that this is not a matter of platform - both are Debian-packages (Linux). Also note that the names are, beside the version suffix, always the same between versions (2.8 vs 2.9), so the ordering should in fact apply to all releases. How about the possibility to define regex's, with user-defined ordering and a fallback to alphanumeric ordering?

Thanks
Tycho
